crates/optimism/chainspec/src/lib.rs

```diff
@@ -368,8 +499,10 @@ mod tests {
 
     #[test]
     fn op_mainnet_forkids() {
+        let op_mainnet = OpChainSpecBuilder::optimism_mainnet().build();
+        let _ = op_mainnet.genesis_hash.set(OP_MAINNET.genesis_hash.get().copied().unwrap());
```

Comment:
is this required?

Reply:
setting the genesis hash is required for the test to pass, although I was thinking that this should probably be set in `OpChainSpecBuilder::optimism_mainnet()`. I didn't do it because `ChainSpecBuilder` leaves the `genesis_hash` field empty by default in the `build` function so wasn't sure if this was something we wanted to update or not.

Reply:
I see why that's the case...